app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.parkingsystem.mvp.view
```

Comment:
this class should be in a base package inside the view package

Reply:
Done 👍